### PR TITLE
Add linting and code coverage

### LINKED ISSUE #2: Add linting to yupana repository
## Feature description:
Add linting to yupana repository

### LINKED ISSUE #3: Add unit test with tox to yupana repository
## Feature description:
Add unit test with tox to yupana repository

### PR DESCRIPTION
**Pull request by kholdaway**
Closes #2 
Closes #3

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    # Don't complain if tests don't hit defensive assertion code:
+    raise AssertionError
+    raise NotImplementedError
+
+    # Don't complain if non-runnable code isn't run:
+    if 0:
+    if __name__ == .__main__.:
+
+ignore_errors = True
```
Comment (abaiken) [inline .coveragerc]: Should there be a `\n` at the end of this file?

**Review by abaiken — CHANGES_REQUESTED** [review]
Left comments & a question. Looking good so far!

File: README.rst [hunk 2]
```diff
@@ -21,6 +21,55 @@ To get started developing against Koku first clone a local copy of the git repos
...
+
+To run a local dev Django server you can use ::
+
+    make run-migrations
+    make serve
+
+Testing and Linting
+-------------------
+
+Koku uses tox to standardize the environment used when running tests. Essentially, tox manages its own virtual environment and a copy of required dependencies to run tests. To ensure a clean tox environment run ::
```
[the remark this answers is not in the document]
Comment (kholdaway) [reply README.rst]: 😂

File: README.rst [hunk 3]
```diff
@@ -21,6 +21,55 @@ To get started developing against Koku first clone a local copy of the git repos
```
[the remark this answers is not in the document]
Comment (kholdaway) [reply README.rst]: 😂

**Review by abaiken — APPROVED** [review]
LGTM 👍

**Pull request merged**